src/app/image-loader/image-loader.component.ts

```diff
@@ -255,6 +276,14 @@ export class ImageLoaderComponent implements OnInit, AfterViewInit, OnDestroy {
     this.preloadSrc = '';
     this.loaded = true;
   }
+  /**
+   * When the main `img` element has loaded
+   *
+   * @memberof ImageLoaderComponent
+   */
+  public onImageLoad(image: ResponsiveImage): void {
+    this.imageLoaded.emit(image);
```

Comment:
I was thinking do we need to emit the image load event when the placeholder is loaded? is it useful?